tests/end2end/test_public_logging_in.py

```diff
@@ -25,6 +25,46 @@ def test_can_log_in_returns_200(user, testapp):
     assert res.status_code == 200
 
 
+def test_user_must_approve_tos_on_login_if_unset(user, testapp):
+    """After login, the user is requested to approve ToS if not done already."""
+    user.tos_approved_at = None
+    user.save()
+    # Goes to pages that requires authentication and gets redirected to login page.
+    res = testapp.get('/collections/?foo=1&baz=2').follow()
+    # Fills out login form.
+    username_with_different_casing = user.email.upper()
```

Comment:
Is there a special reason for this that we should document? I would assume it's from a copied test and thus covered elsewhere? (No need to change if that's the case.)

Reply:
Not really.. Just trying to regularly emphasize that emails are not validated case sensitive. :)

Reply:
@mxtthias Line 19 above contains the same variable assignment with an inline comment. Should I remove the duplicate here?

Reply:
Nah, I'd say we keep it as is. :)